src/main/java/io/codelex/loops/practice/FizzBuzz.java

```diff
@@ -0,0 +1,28 @@
+package io.codelex.loops.practice;
+
+import java.util.Scanner;
+
+public class FizzBuzz {
+    public static void main(String[] args) {
+        Scanner input = new Scanner(System.in);
+        System.out.println("Enter max value?");
+        int maxValue = input.nextInt();
+
+        for (int i = 1; i <= maxValue; i++) {
+            if (i % 20 == 0) {
```

Comment:
Šis īsti nestrādās pareizi - pie skaitļa 20 viņam vajadzētu izvadīt Buzz, bet vinš izvadīs skaitli 20 un tad jaunu rindiņu.
Padoms: atstāj tikai tos if-else, kuri ir sākot no 16 rindas un tikai pēc tam atsevišķa IF pārbaudi vai i % 20 == 0 un, ja ir - izvadi jaunu rindas simbolu ar System.out.println()